tools/lsp/preview/ui/brushes.rs

```diff
@@ -147,7 +155,7 @@ pub fn create_brush(
             i_slint_core::graphics::RadialGradientBrush::new_circle(stops.drain(..)),
         ),
         ui::BrushKind::Conic => slint::Brush::ConicGradient(
-            i_slint_core::graphics::ConicGradientBrush::new(stops.drain(..)),
+            i_slint_core::graphics::ConicGradientBrush::new(angle / 360.0, stops.drain(..)),
```

Comment:
why is the angle divided by 360 here but not n the case of the linear gradient two line bellow? that seems inconsistant